### PR TITLE
Add BICC logo and info

### PR DESCRIPTION
**Pull request by watanofs**
https://bscscan.com/tx/0x12c8adb8778530dd85cf9568d7c65cf5513a397c26733a7da78fed7210ffd73a
https://bscscan.com/tx/0x1c34e46b28d1fcedbad3831b63165ba3a9b8bab485622f45512969e3c552eec9

### CODE REVIEW

**Comment by opera0001** [comment]
The agency contract is detected, which has a high risk.

**Comment by watanofs** [comment]
BICC use the openzeppelin upgrade contract. it's the DAO, so we need upgrade BICC contract code as needed by the community. BICC feature was finished, but we need test and keep it stable. then we will revoke admin role. And BICC token had publish 45 days, the market has returned to stability. Now we need let more people know about BICC, so pls help us upload the BICC icon on tokenpocket wallet, tks.

**Comment by opera0001** [comment]
Sorry, the agency contract is very risky. We can't take the risk of collecting the logo for you. The TPT hash value link you submitted can also be used as the next token submission.

**Comment by watanofs** [comment]
1. The *OpenZeppelin upgradeable contract* is a common design pattern in
the industry, and there is even an *EIP1967 *on this. Many well-known
projects in the coin circle are using this set of standards, and many
approved projects in the TokenPocket wallet also use OpenZeppelin
upgradeable contracts.
2. The BICC token has been released for *46 days*, and there is already
*$120k* of liquidity on the *PancakeSwap *exchange, with an overall market
capitalization of *$3.6M*. This is a mature and market-recognized project,
and the project is also preparing to launch the second phase of the NFT
module this week.
3. Once the second phase of the NFT module is stable, we will open source
the contract code of the BICC token, revoke the administrator permissions,
and lock the  liquidity  pool wallet.
In short, BICC is a high-growth project, and we hope that the TokenPocket
wallet can give more attention and help us. We also do our best to assist
you in reviewing our contracts.
Please help us, thank you very much.


   - OpenZeppelin <Using with Upgrades> -
   https://docs.openzeppelin.com/contracts/4.x/upgradeable
   - OpenZeppelin <Proxy> -
   https://docs.openzeppelin.com/contracts/4.x/api/proxy
   - EIP1967 - https://eips.ethereum.org/EIPS/eip-1967


opera0001 ***@***.***> 于2022年5月27日周五 08:38写道：
